lib/symboldatabase.cpp

```diff
@@ -2001,13 +2001,55 @@ Function::Function(const Tokenizer *mTokenizer,
     }
 }
 
+namespace {
+    std::string qualifiedName(const Scope *scope)
```

Comment:
Please use `static` for private functions. Please feel free to use anonymous namespaces for types. It makes it easier to debug with gdb. I can use `break qualifiedName` if it's static but not if you use a anonymous namespace.

Reply:
Done.